flutter_local_notifications/README.md

```diff
@@ -166,6 +167,7 @@ Notification icons should be added as a drawable resource. The example project/c
 
 When specifying the large icon bitmap or big picture bitmap (associated with the big picture style), bitmaps can be either a drawable resource or file on the device. This is specified via a single property (e.g. the `largeIcon` property associated with the `AndroidNotificationDetails` class) where a value that is an instance of the `DrawableResourceAndroidBitmap` means the bitmap should be loaded from an drawable resource. If this is an instance of the `FilePathAndroidBitmap`, this indicates it should be loaded from a file referred to by a given file path.
 
+If the plugin is initialized in the `main()` method make sure that `WidgetsFlutterBinding.ensureInitialized()` is executed before. Otherwise it can happen, that the plugin throws Null Pointer Exceptions. Also see issue [#1364](https://github.com/MaikuB/flutter_local_notifications/issues/1364) for more information.
```

Comment:
This doesn't have to do with plugin being initialised. This is to do with trying to use platform channels in general so it could help with calling any method that uses platform channels.

Reply:
Oh, this is from another issue where I created a PR weeks ago. Seems like it was accidentally merged into this one. Should I keep it in or remove it?

Reply:
If you could remove it then that would be preferred. Regarding submission of PRs, a tip I learnt that can help in situations like this is to create a branch in your fork instead of having it all on the main/master. That allows the latter to be kept up to date more easily and can separate features/fixes per branch

Reply:
done.